worlds/smw/Options.py

```diff
@@ -1,6 +1,7 @@
 from dataclasses import dataclass
 
-from Options import Choice, Range, Toggle, DeathLink, DefaultOnToggle, OptionGroup, PerGameCommonOptions
+from Options import Choice, Range, Toggle, DeathLink, DefaultOnToggle, OptionGroup, PerGameCommonOptions, PlandoTexts
+from worlds.smw.Names.TextBox import text_table
```

Comment:
Import needs to be relative and not absolute
```suggestion
from Names.TextBox import text_table
```